src/utils/getNextStepper/getNextStepper.ts

```diff
@@ -0,0 +1,25 @@
+import type { TreeNode } from "@/types/TreeNode";
+
+const getNextStepper = (restArray?: TreeNode[]) => {
+  if (!restArray?.length) {
+    return 0;
+  }
+
+  let stepper = 0;
+
+  for (let i = 0; i < restArray.length; i += 1) {
```

Comment:
Pr au top fréro 💪 Ta encore fait du super job vraiment je suis happy !
Juste ici si tu peu me faire quelque chose de plus déclaratif avec un jolie petit `array.find()` ou autres  pour virer le `for` ça serait perfect